components/merkledb/README.md

```diff
@@ -0,0 +1,84 @@
+# exonum-merkledb
```

Comment:
Should we make the first letter capital?

Reply:
I don't know if we have a naming convention for this, but all of our readme files start with lowercase crate name.

Reply:
Except for [Exonum](https://github.com/exonum/exonum/blob/master/README.md) and [Exonum core library](https://github.com/exonum/exonum/blob/master/exonum/README.md) I guess.

As for me it looks strange that in the title `exonum-merkledb` is written in lowercase and just a little below you can see `Exonum-merkledb`.

Reply:
Now I think it's better to use library name here without `exonum` prefix as it will be positioned as a general purpose library.
@aleksuss what do you think?

Reply:
I'd name it `Exonum MerkleDB`.